### DISCUSSION TITLE
Workflow on the TUI design?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [tznind/TerminalGuiDesigner#209] [General]
Hi, i just noticed theat, i have this frame that starts visible=false (runtime) but in design time its always visible, is there a better way to design my TUI where these frames can lay above the "main view"?

![image](https://github.com/gui-cs/TerminalGuiDesigner/assets/12295022/81749bfc-5bcb-4e35-9522-228c0114ff75)

**Comment by tznind** [comment]
I think the easiest way to do this would be to use `New...` to create a new view e.g. called `SettingsView` and choose `Window` as the base class.

You could then launch this second view from your first with `Application.Run<SettingsView>();` e.g.

```csharp
namespace YourNamespace {
    using System;
    using Terminal.Gui;
    
    public partial class MyView {
        
        public MyView() {
            InitializeComponent();

            editMeMenuItem.Action = ShowSettings;
        }

        private void ShowSettings()
        {
            Application.Run<SettingsView>();
        }
    }
}

```
You can quit the popup and return to the main app with Ctrl+Q 

Also I think you want to make sure you have your Window be

```csharp
Modal = true;
```

**Reply by GronbergInfo** [reply]
Thank you, forgot to reply earlier.
